pom.xml

```diff
@@ -524,6 +524,17 @@
         <artifactId>libthrift</artifactId>
         <version>${thrift-version}</version>
       </dependency>
+      <dependency>
+        <!-- override litelinks' versions with newer to avoid CVE -->
+        <groupId>org.apache.zookeeper</groupId>
+        <artifactId>zookeeper</artifactId>
+        <version>3.7.2</version>
```

Comment:
wouldn't be better move these version to the _<properties>_ tag?

Reply:
Now updated